### ISSUE TITLE
[Bug]: Non US/ANSI keyboard not being able to change playback speed

### ISSUE THREAD

**Issue by Petingoso** [ThaUnknown/miru#236]
### Preflight checklist

- [X] I have searched the [issue tracker](https://github.com/ThaUnknown/miru/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [frequently asked questions](https://github.com/ThaUnknown/miru/blob/master/docs/faq.md) for a solution that fixes this problem, without success.
- [X] I have checked that I'm using the [latest stable](https://github.com/ThaUnknown/miru/releases/latest) version of the app.

### What app version are you using?

v3.5.3

### What operating system version are you using?

Linux

### Expected Behavior

In the Player the expected behavior is to press [ or ] and the playback speed in the player to increase.

### Actual Behavior

Currently, on wayland/wlroots/sway, I'm unable to input [ or ] to the program to change it.

I have a portuguese ISO keyboard and i require AltGr+8/9 to input it which the program doesn't seems to catch.

I tried to use a program to input the characters for me, but for some reason it does not also seem to work, despite being fine on other apps (mpv for example, works on both real and simulated input).

I propose either manual buttons or to review the method on how shortcuts are detected.

### Screenshots

![image](https://user-images.githubusercontent.com/92183955/210649556-0e73b4ed-7d07-402a-b69f-f4ab741d50e2.png)
![image](https://user-images.githubusercontent.com/92183955/210650030-237e71f1-6f16-499f-b77d-92b073802a10.png)

**Comment by ThaUnknown** [comment]
this is an issue with your operating system, the [keyboard api](https://developer.mozilla.org/en-US/docs/Web/API/Keyboard_API) supports layouts, which is why the letters on the keyboard ui arent labeled, because they work regardless of the user's keyboard layout

this is an issue with either linux, your distro, your distro config or chromium

**Comment by ThaUnknown** [comment]
note that they keybind isnt actually [ or ], just whatever keys are in that place, anyways this works fine on windows with dworak etc so I won't be going out of my way to fix some minor linux distro

**Comment by Muril-o** [comment]
I can confirm this works just fine on sway since I'm a sway user. I also use a different keyboard layout (ABNT2) and no problems so far as long as I think a bit where each key should be relative to the US layout.

> the [keyboard api](https://developer.mozilla.org/en-US/docs/Web/API/Keyboard_API) supports layouts, which is why the letters on the keyboard ui arent labeled, because they work regardless of the user's keyboard layout

Adding couple cents here, this is not entirely true. The layout format itself that is shown on screen doesn't change because its immutable, more specifically writing system keys are immutable. Some keyboard layouts (like ABNT2) simply have more keys which aren't shown in the displayed layout, but are valid writing system keys. This means you cannot bind some of the keys your keyboard has regardless of the Keyboard API working.
Take for example the default keys for playback speed change, lets assume they are bound to the '[' ']' keys as usual (US layout) on the second row. Me on ABNT2 actually have to press a key on the 2nd row (for slowdown) and one on the 3rd row (for speeding up), even tho virtually they are both bound to keys on the 2nd row. (regardless of OS in case you're wondering, same happens on windows)

Keyboard layouts are too complicated because they can either be remapped on software (which is what we do on sway for using dvorak on a qwerty keyboard), or remapped on the keyboard itself which they just switch the activation keys with the default qwerty layout so you can type without needing a software. Some custom keyboards even require a software on top of the keyboard itself because instead of switching activations with qwerty they simply send the default ones and its later corrected by said software.

TL;DR Layouts and keyboards are a mess. Specially when you consider custom made keyboards which don't abide by the conventions, so I'd say fitting every single layout inside Miru is not worth the effort as you can just rebind some keys keeping in mind the displayed layout (which at this point it should be better referred as "writing system keys" instead of "layout") and get used to it with time.

Unless you want to change the default displayed one to a more inclusive displayed layout, but that would just confuse the majority of like 95% users who use the US layout. imo I believe miru by using writing system keys the way it is right now based on the US layout is already general enough for me to use and the minority can get around it by remapping keys or just thinking a bit on their layout relative to the US one.
